woorizip-common/src/main/kotlin/common/exception/ErrorCode.kt

```diff
@@ -2,25 +2,9 @@ package common.exception
 
 interface ErrorCode {
     fun getCode(prefix: String, ordinal: Int): String {
-        return when (val number = ordinal + 1) {
-            in UNITS_MIN..UNITS_MAX -> {
-                "$prefix-00$number"
-            }
-
-            in TENS_MIN..TENS_MAX -> {
-                "$prefix-0$number"
-            }
-
-            else -> {
-                "$prefix-$number"
-            }
-        }
+        val numberCode = "%03d".format(ordinal)
```

Comment:
```suggestion
        val numberCode = "%03d".format(ordinal + 1)
```
**ordinal** 사용이 좋지 않다고 하는데, **ordinal**을 사용하는 만큼 **enum** 순서 주의하면서 만들면 문제 없지 않을까요 🧐
현직 개발자는 어떻게 생각할지 궁금하네요...

Reply:
그러네요 자바 공식 문서에서 쓰지 말라니까.. 쓰지말아야죠 ☹️